Nodejs-Open-Collective.md

```diff
@@ -0,0 +1,35 @@
+# Node.js Open Collective
+
+The project maintains an Open Collective that is used to accept donations.
+
+The funds are to be used for specific efforts as decided by the TSC and
+documented on this page.
+
+## Account details
+
+The account is manged by the TSC. The user id and password
+are shared with TSC members through 1password.
+
+## Disbursements - management
+
+Potential disbursements will be agreed through TSC discussion
+and before approval, documented in an PR in the
+[TSC repository](https://github.com/nodejs/TSC) which adds
+the intended use and funding envelope to this page. The disbursement
+is considered approved once TSC consensus in the PR has
+been reached.
```

Comment:
wdyt about adding a requirement for taking a vote if there's an objection? The idea would be to make it harder for TSC members to take controversial money decisions.
```suggestion
the intended use and funding envelope to this page. The disbursement
is considered approved if there's no objections from a project member
in 72 hours, and at least one approval from a TSC voting member.
Objections can be overridden with a TSC vote.
```

Reply:
@aduh95 I assume you mean an objection from a non-TSC member as a vote would already be required if there was an objection from a TSC member.

I'd be ok with that.

Reply:
I'm ok with what is proposed, but I would propose something different. 

> Disbursements are approved in TSC meetings with quorum. In case of any objections, a TSC vote would be required.

Reply:
@mcollina I think using quorum in a meeting does not align with making sure everybody has an opportunity to comment or object. I am happy to add the the clause about objections, but I think we should clarify in terms of "objections by who". Options could include:

* TSC members
* An members of the Node.js organization
* core collaborators (although I think this is not broad enough)
* Anybody (I presonally think this is too broad)


@mcollina, @aduh95 any other options than those and thoughts on which we should specify?